typescript/packages/plugins/pumpfun/src/token.ts

```diff
@@ -0,0 +1,59 @@
+import { Connection, Keypair, PublicKey, SystemProgram, Transaction } from '@solana/web3.js';
+import type { SolanaWalletClient } from '@goat-sdk/core';
+
+// Pump.fun program ID on Solana
+const PUMP_FUN_PROGRAM_ID = new PublicKey('6EF8rrecthR5Dkzon8Nwu78hRvfCKubJ14M5uBEYfBH1');
```

Comment:
where did you get this from?

From what i can see online, its this 6EF8rrecthR5Dkzon8Nwu78hRvfCKubJ14M5uBEwF6P